public/js/controllers/index.js

```diff
@@ -71,12 +72,17 @@ angular.module('mean.system').controller('IndexController', [
         $http.post('/api/auth/signup', $scope.user)
           .then(
             () => {
-              $location.path('/confirmaccount');
+              // $location.path('/confirmaccount');
```

Comment:
Do same here and every other place we have commented out code @Daramola98

Reply:
Done @fortunee